index.html

```diff
@@ -424,36 +424,123 @@ <h4>Double-sided ruby, when both read aloud</h4>
-            In some cases, it causes mistakes in deciding whether <span lang="ja">へ</span> should be read aloud 
-            as <span lang="ja">え</span> (E) or <span lang="ja">へ</span> (HE) and 
-            whether <span lang="ja">は</span> should be read aloud as <span lang="ja">わ</span> (WA) or <span lang="ja">は</span> (HA). 
-            This is because the morphological analysis does not work properly and pronunciation dictionaries 
-            for compound words cannot be used, as kana characters are used instead of CJK ideographic characters.
-            As an example, consider <span lang="ja">今後は<ruby><rb>発展</rb><rt>はってん</rt></ruby></span>. 
-            Text-to-speech of <span lang="ja">今後は発展</span> typically works fine but that of <span lang="ja">今後ははってん</span> does not. 
-            The first occurrence of <span lang="ja">は</span> should be read aloud as <span lang="ja">わ</span> (WA) 
-            but is mistakenly read aloud as <span lang="ja">は</span> (/ha/).
-          </p>
-          <p>
-            Even when this option is used, it might be wise to ignore furigana-added-for-enhanced-accessibility but rely on ruby bases.
-          </p>
-          <p>
-            If furigana is assigned only for the first occurrence of a word, 
-            there is a risk that the first occurrence and the others are read aloud differently.
+	    Even native Japanese speakers may easily assume, without
+	    thorough consideration, that the option of reading only ruby
+	    annotations aloud will provide reasonable results. However,
+	    this is not always the case.
           </p>
+
+	  <section>
+	    <h5>Incorrect pitch accent</h5>
+
+	    <p>Each hiragarana character represents a mora (a basic timing
+	      unit in phonology), which is typically a single vowel or a
+	      consonant followed by a single vowel.  The same sequence of
+	      moras may mean different words depending on the pitch accent.
+	      For example, both <span lang="ja">雨</span> (rain) and
+	      <span lang="ja">飴</span> (candy) consists of the same moras:
+	      <span lang="ja">あ</span> and <span lang="ja">め</span>.
+	      However, the first mora in <span lang="ja">雨</span> has a
+	      high pitch, and the second has a low pitch;
+	      <span lang="ja">飴</span> has the opposite pitch accent.</p>
+
+	    <p>Reading aloud ruby annotations rather than ruby bases often
+	      leads to incorrect pitch accent. As an example, consider
+	      <span lang-"ja"><ruby><rb>雨</rb><rt>あめ</rt></ruby>が好き</span>
+	      (I like rain) and <span lang-"ja"><ruby><rb>飴</rb><rt>あめ</rt></ruby>が好き</span>
+	      (I like candy). In both cases, reading aloud ruby annotations
+	      rather than ruby bases implies that the TTS engine will receive
+	      <span lang-"ja">あめが好き</span> and create the same result.</p>
+
+	    <p>A similar example is <span lang-"ja"><ruby><rb>牡蠣</rb><rt>かき</rt></ruby>を食べる</span>
```

Comment:
I could not get how this is simple. consisted with two moras are the same....??

Reply:
If you do not understand, non-native speakers will have serious problems.  Which part of my description is difficult?

Reply:
I could understand the example itself, but quite not sure why it is simpler. consisted of two mora (or two hiragana) for ruby annotation is the same as the above example.

Reply:
I just wonder if you misunderstood the word "similar"?